nexus/db-model/src/schema_versions.rs

```diff
@@ -29,6 +29,7 @@ static KNOWN_VERSIONS: Lazy<Vec<KnownVersion>> = Lazy::new(|| {
         // |  leaving the first copy as an example for the next person.
         // v
         // KnownVersion::new(next_int, "unique-dirname-with-the-sql-files"),
+        KnownVersion::new(63, "remove-producer-base-route-column"),
```

Comment:
I don't see the migration files for this directory - missing in the commit maybe?

Reply:
`git add -u` strikes again!

Reply:
Added in a54ce97ac